javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -722,6 +757,9 @@ protected SymbolResolver getSymbolResolver() {
     public static final DataKey<SymbolResolver> SYMBOL_RESOLVER_KEY = new DataKey<SymbolResolver>() {
     };
 
+    public static final DataKey<LineEnding> LINE_ENDING_KEY = new DataKey<LineEnding>() {
```

Comment:
Just noticed this. It might be able to be made private -- originally there wasn't an accessor method on Node. Maybe we can also add in a mutator / setter method.

Reply:
resolved -- leaving as public